index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,400;0,600;1,100&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="./main.css">
+    <title>Pascal</title>
+</head>
+
+<body>
+    <header class="header">
+        <div class="navigation">
+            <div class="logo">
+                <h1 class="logo-text">Pascal</h1>
+            </div>
+            <div><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" class="logo-text" viewBox="0 0 24 24">
+                    <path d="M24 6h-24v-4h24v4zm0 4h-24v4h24v-4zm0 8h-24v4h24v-4z" />
+                </svg></div>
```

Comment:
- [ ] The hamburger button on the right should be inside a `nav` tag. :writing_hand: